### PR TITLE
[DC-970] Uncomment searchConcept and make domainId an int

### PR DESCRIPTION
**Pull request by s-rubenstein**

### CODE REVIEW

**Review by evansuslovich — APPROVED** [review]
Looks good!

File: src/test/java/bio/terra/integration/DataRepoFixtures.java [hunk 1]
```diff
@@ -1885,7 +1885,7 @@ public SnapshotBuilderGetConceptsResponse getConcepts(
   }
 
   public SnapshotBuilderGetConceptsResponse searchConcepts(
-      TestConfiguration.User user, UUID datasetId, String domainId, String searchText)
+      TestConfiguration.User user, UUID datasetId, Integer domainId, String searchText)
```
Comment (pshapiro4broad) [inline src/test/java/bio/terra/integration/DataRepoFixtures.java]: If this parameter can't be null, it should be declared as `int`, to make the intent of the code clear
```suggestion
      TestConfiguration.User user, UUID datasetId, int domainId, String searchText)
```

**Review by pshapiro4broad — APPROVED** [review]
Looks good. It's kind of odd that the GCP test is a connected test and the azure one is an integration test.

**Review by snf2ye — APPROVED** [review]
Thanks!

**Pull request merged**